### PR TITLE
Update CLI docs

### LINKED ISSUE #110: Doc how to get autocomplete support
PR: https://github.com/PegaSysEng/teku/pull/2234

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull request checklist

Use the following list to make sure your PR fits the Teku doc quality standard.

### Before creating the pull request

Make sure that:

- [x] you have read the [contribution guidelines](https://github.com/ConsenSys/doc.common/wiki/Contributing-to-Documentation).
- [x] you have [tested your changes locally](https://github.com/ConsenSys/doc.common/wiki/MkDocs-And-Custom-Markdown-Guide#preview-documentation-site-locally) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you have fixed all the issues raised by the tests, if any.
- [x] you have verified the rendering of your changes on
  [ReadTheDocs.org PR preview](https://github.com/ConsenSys/doc.common/wiki/MkDocs-And-Custom-Markdown-Guide#preview-the-doc-site-for-your-pr-on-readthedocscom)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Update CLI docs for consistency. Add subcommands overview page, and doc how to use autocomplete support.

## Issue fixed

fixes #110 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

## Testing

https://pegasys-teku--322.com.readthedocs.build/en/322/Reference/CLI/CLI-Syntax/

### CODE REVIEW

**Review by ajsutton — APPROVED** [review]
LGTM

**Review by rolandtyler — APPROVED** [review]
lgtm

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -4,40 +4,50 @@ description: Teku command line interface reference
 
 # Teku command line
 
-This reference describes the syntax of the Teku Command Line Interface (CLI) options and subcommands.
+This reference describes the syntax of the Teku command line interface (CLI) options.
 
 !!! important
```
Comment (bgravenorst) [inline docs/Reference/CLI/CLI-Syntax.md:9]: This was added when Teku was still in the early stages of development. Perhaps it's time to remove it?

**Review by bgravenorst — APPROVED** [review]
lgtm

**Pull request merged**